### DISCUSSION TITLE
typescript for docs

### DISCUSSION THREAD

**Discussion by mbnoimi** [quasarframework/quasar#14767] [Ideas / Proposals] [answered]
Current docs (2.10.1) examples doesn't support typescript.
May you please support typescript in the docs.

**Comment by rstoenescu** [comment, marked as answer]
The examples need to be as inclusive as possible. Not everyone uses Typescript. Some use plain JS, some use UMD. They all have to be accounted for. The current state of the examples allows for everyone to fill in the blanks regardless of what they chose to use.

**Reply by mbnoimi** [reply]
Thanks a lot for the quick response.
I'm one of those who hate Javascript and tend to use Typescript in all my projects, One of the essential things I loved in Quasar is TS support so these examples don't fill the blanks they create headache in all around. Maybe because I came from C++ background. But most modern frameworks uses TS even the old ones rewrite their codes into TS.
Any way, I know my reply isn't valuable because it will not convince you to take this decision but I just want to tell you (as a maintainer) that there are some people really hate JS and thank god there are something replace it.
Thanks a lot for the great project.

**Reply by 54mu3l** [reply]
@mbnoimi If you have trouble writing Typescript (with Quasar) then I would recommend checking the Vue docs. There is a whole section about Typescript. With this you should be completely fine using Typescript with Quasar.
